From 5a0d1a3177dbc8785f655b5d0a9a6861cc0f1625 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 11 Apr 2021 05:50:13 +0000
Subject: [PATCH] WIP: dt-bindings: arm: amlogic: add support for Minix NEO-U1

The Minix NEO-U1 is an Android STB based on the Amlogic P200 (GXBB)
reference design with an S905 chip.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 6f3c43ae92c2..851aff819051 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -58,6 +58,7 @@ properties:
               - amlogic,p201
               - friendlyarm,nanopi-k2
               - hardkernel,odroid-c2
+              - minix,neo-u1
               - nexbox,a95x
               - videostrong,kii-pro
               - wetek,hub
